src/components/Counter/Counter.vue

```diff
@@ -0,0 +1,73 @@
+<!--
+ - @copyright Copyright (c) 2019 Marco Ambrosini <ma12co@pm.me>
+ -
+ - @author Marco Ambrosini <ma12co@pm.me>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<template>
+
+	<!-- counter -->
+	<li :class="{highlighted: isMention}"
+		class="app-navigation-entry-utils-counter">
+		<span v-if="!isMention">{{ item.utils.counter }}</span>
+		<span v-if="isMention">@ {{ item.utils.mentionsCounter }}</span>
+	</li>
+
+</template>
+
+<script>
+
+export default {
+	name: 'Counter',
+	props: {
+		isMention: {
+			type: Boolean,
+			required: true
+		},
+		item: {
+			type: Object,
+			required: true
+		}
+	}
+}
+
+</script>
+
+<style lang="scss" scoped>
+
+	.app-navigation-entry-utils-counter {
+		overflow: hidden;
+		text-align: right;
+		font-size: 9pt;
+		line-height: 44px;
```

Comment:
https://github.com/nextcloud/nextcloud-vue/blob/ba1d4f83e53315f57ba099e6e6d0282d9c950f31/src/assets/variables.scss#L26
```suggestion
		line-height: $clickable-area;
```